python/arcticdb/version_store/_store.py

```diff
@@ -2226,6 +2199,84 @@ def get_info(self, symbol: str, as_of: Optional[VersionQueryInput] = None) -> Di
             "date_range": self.get_timerange_for_symbol(symbol, vit.version),
         }
 
+    def get_info(self, symbol: str, as_of: Optional[VersionQueryInput] = None) -> Dict[str, Any]:
+        """
+        Returns descriptive data for `symbol`.
+
+        Parameters
+        ----------
+        symbol : `str`
+            symbol name
+        as_of : `Optional[VersionQueryInput]`, default=None
+            See documentation of `read` method for more details.
+
+        Returns
+        -------
+        `Dict[str, Any]`
+            Dictionary containing the following fields:
+
+            - col_names, `Dict`
+            - dtype, `List`
+            - rows, `int`
+            - last_update, `datetime`
+            - input_type, `str`
+            - index_type, `index_type`
+            - normalization_metadata,
+            - type, `str`
+            - date_range, `tuple`
+        """
+        version_query = self._get_version_query(as_of)
+        vit, desc = self.version_store.read_descriptor(symbol, version_query)
+        return self._process_info(symbol, vit, desc, as_of)
+
+    def get_batch_info(
+        self, symbols: List[str], as_ofs: Optional[List[VersionQueryInput]] = None
+    ) -> List[Dict[str, Any]]:
+        """
+        Returns descriptive data for a list of `symbols`.
+
+        Parameters
+        ----------
+        symbols : `str`
+            symbols: `List[str]`
+                List of symbols to read the metadata for. Elements should not be duplicated.
```

Comment:
What does this mean? 

> Elements should not be duplicated.

Can you not return metadata for different versions of the same symbol?

Reply:
Sorry, We do, yes. I will remove the comment and add the corresponding test